src/deepsparse/server/openai_server.py

```diff
@@ -194,7 +198,7 @@ async def create_chat_completion(raw_request: Request):
                 )
                 choices.append(choice_data)
 
-            num_prompt_tokens = len(final_res.prompt_token_ids["input_ids"])
+            num_prompt_tokens = len(final_res.prompt_token_ids["input_ids"][0])
```

Comment:
`prompt_token_ids` is now a List[List[int]]? maybe we should note that batching isn't supported and this is why `[0]` is valid

Reply:
it's the format being returned by the pipeline but yah, I can make note.